pages/Addons.md

```diff
@@ -84,6 +84,7 @@ SpiritsUnchained | JustAHuman_xD | <details><summary>expand</summary>Spirits Unc
 Cultivation | Sefiraat / J3fftw | <details><summary>expand</summary>Cultivation is an addon that works to emulate elements of Mystical Agriculture, Cooking for BlockHeads and HarvestCraft in a vanilla experience. Using 1.19.4 features to produce spectacular modded-style blocks. We offer over 90 plants that generate vanilla resources like ores, mobs drops etc., over 90 bushes and custom tree builds that can be harvested for produce used for Cooking. Regards to cooking, you can build you own kitchen in a desgin you see fit and cook freely resulting in foods with effects such as healing, potion effects, teleportation, mob spawning and more!</details> | [Readme](https://github.com/Sefiraat/Cultivation/blob/main/README.md) | [Github](https://github.com/Sefiraat/Cultivation) | [Builds](https://thebusybiscuit.github.io/builds/Sefiraat/Cultivation/main/)
 Gastronomicon | SchnTgaiSpock | <details><summary>expand</summary>A Slimefun addon that adds hundreds of new foods to craft and eat</details> | [Readme](https://github.com/SchnTgaiSpock/Gastronomicon/blob/master/README.md) | [Github](https://github.com/SchnTgaiSpock/Gastronomicon) | [Builds](https://thebusybiscuit.github.io/builds/SchnTgaiSpock/Gastronomicon/master/)
 SmallSpace | CAPS123987 | <details><summary>expand</summary>This addon helps you when you have small space, just place a block, right click it and you will be teleported to your space, you can also add members using /smallspace addMember</details> | [Readme](https://github.com/CAPS123987/SmallSpace/blob/master/README.md) | [Github](https://github.com/CAPS123987/SmallSpace) | [Builds](https://thebusybiscuit.github.io/builds/CAPS123987/SmallSpace/master/)
+BetterNuclearGenerator | CAPS123987 | <details><summary>expand</summary>This addon adds better nuclear reactor - realistic</details> | [Readme](https://github.com/CAPS123987/Better-Nuclear-Generator/blob/master/README.md) | [Github](https://github.com/CAPS123987/Better-Nuclear-Generator) | [Builds](https://thebusybiscuit.github.io/builds/CAPS123987/Better-Nuclear-Generator/master/)
```

Comment:
The description doesn't really read too well, so I have fleshed it out in the suggestion below, if you like it!

```suggestion
BetterNuclearGenerator | CAPS123987 | <details><summary>expand</summary>This addon adds a more powerful, and more realistic, nuclear reactor to use for more versatile power generation.</details> | [Readme](https://github.com/CAPS123987/Better-Nuclear-Generator/blob/master/README.md) | [Github](https://github.com/CAPS123987/Better-Nuclear-Generator) | [Builds](https://thebusybiscuit.github.io/builds/CAPS123987/Better-Nuclear-Generator/master/)
```